ansible/group_vars/nimbus.holesky.yml

```
@@ -171,6 +176,13 @@ nimbus_era_files_network: '{{ beacon_node_network }}'
 nimbus_era_files_db_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/data/db'
 nimbus_era_files_nclidb_path: '/data/beacon-node-{{ beacon_node_network }}-unstable/repo/build/ncli_db'
 
+# RPC-snooper
+rpc_snooper_service_name: 'rpc-snooper-{{ beacon_node_network }}-{{ "%02d"|format(idx|int+1) }}'
+rpc_snooper_cont_port:   '{{ 8700  + (idx|int) + 1 }}'
```

It should be defined at the top, same way `exec_layer_authrpc_port` is.
```suggestion
rpc_snooper_cont_port:   '{{ exec_layer_snooper_port }}'
```